src/editors/data/redux/app/selectors.js

```diff
@@ -61,7 +61,7 @@ export const analytics = createSelector(
     module.simpleSelectors.courseId,
   ],
   (blockId, blockType, courseId) => (
-    {blockId, blockType, courseId}
+    { blockId, blockType, courseId }
```

Comment:
The analytics we want to send is set up here. This can be changed in the future if we want more than just blockId, blockType and courseId.

Reply:
Maybe rename this to analyticsData or something that better describes that this is a payload to be used, not the actual analytics system itself? does that make sense?